doc/howto/images_copy.md

```diff
@@ -1,100 +1,88 @@
 
-### File on a remote web server
+```{note}
+To copy the image to your local image store, specify `local:` as the target remote.
+```
 
-As an alternative to running a full image server only to distribute a
-single image to users, LXD also supports importing images by URL.
+See `lxc image copy --help` for a list of all available flags.
+The most relevant ones are:
 
-There are a few limitations to that method though:
+`--alias`
+: Assign an alias to the copy of the image.
 
-- Only unified (single file) images are supported
-- Additional HTTP headers must be returned by the remote server
+`--copy-aliases`
+: Copy the aliases that the source image has.
 
-LXD will set the following headers when querying the server:
+`--auto-update`
+: Keep the copy up-to-date with the original image.
 
-- `LXD-Server-Architectures` to a comma-separated list of architectures the client supports
-- `LXD-Server-Version` to the version of LXD in use
+`--vm`
+: Copy an image that can be used to create virtual machines.
 
-And expects `LXD-Image-Hash` and `LXD-Image-URL` to be set by the remote server.
-The former being the SHA256 of the image being downloaded and the latter
-the URL to download the image from.
+## Import an image from files
 
-This allows for reasonably complex image servers to be implemented using
-only a basic web server with support for custom headers.
+If you have image files that use the required {ref}`image-format`, you can import them into your image store.
 
-On the client side, this is used with:
+There are several ways of obtaining such image files:
 
-    lxc image import URL --alias some-name
+- Exporting an existing image (see {ref}`images-manage-export`)
+- Building your own image using `distrobuilder` (see {ref}`images-create-build`)
+- Downloading image files from the [image server](https://images.linuxcontainers.org/images/) (note that it is usually easier to {ref}`use the remote image <images-remote>` directly instead of downloading it to a file and importing it)
 
+### Import from the local file system
 
-## Import images
-You can import images, that you:
+To import an image from the local file system, use the `lxc image import` command.
+This command supports both {ref}`unified images <image-format-unified>` (compressed file or directory) and {ref}`split images <image-format-split>` (two files).
 
-- built yourself (see [Build Images](#build-images)),
-- downloaded manually (see [Manual Download](#manual-download))
-- exported from images or containers (see [Export Images](#export-images) and [Create Image from Containers](#create-image-from-containers))
+To import a unified image from one file or directory, enter the following command:
 
-#### Import container image
+    lxc image import <image_file_or_dir> [<target_remote>:]
 
-Components:
+To import a split image, enter the following command:
 
-- lxd.tar.xz
-- rootfs.squashfs
+    lxc image import <metadata_tarball> <rootfs_tarball> [<target_remote>:]
```

Comment:
Can you add `_path` to these to avoid any confusion around what these arguments are (i.e they are not the file contents themselves).